### ISSUE TITLE
Document correct config in SCSS getting started guide

### ISSUE THREAD

**Issue by Andy-QT** [stylelint/stylelint#8317]
### What minimal example or steps are needed to reproduce the bug?

```css
:root {
	--color: grey;
}

$fontcolor: var(--color);
$bordercolor: var(--color);

.container {
	color: $fontcolor;
	border: 1px solid $bordercolor;

}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "extends": [
    "stylelint-config-standard"
  ]
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIA6A7eAnCEAuACYqS2AtMZADYTry4Dm6MAngNyoC+hKAJAGYQrYKVGgDcAhugAUpIegCUrbgCMqsdAGEIlarnFSZWqgs4A6SALEBLFDHT5Cgwzt79H2xURXo1NAIwAHAA9cAGctSyhcLi81TXdONhAAGhAeS3IYADkxAFs4RBhA3P8MsxCQ5PB+NNoEEAIUXFwkEELsGBQoEJaaAG1UJqaWkOxGDPJrEnMa4hGxTokoFoHcAF12SunLWgAxKhyxbDqAKzCUSth-CsQGoZARsZgJgR7mkHJDmBGWpJXh0fGkzI1W2s2w8ygi1eLQ+7W+yBQiTYQA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

No error instead of Cannot parse property value "1px solid $borderColor" for property "border"

### What actually happened?

The following problem was reported: 

11:10-32 error Cannot parse property value "1px solid $borderColor" for property "border"
[(declaration-property-value-no-unknown)](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown)

### Do you have a proposal to fix the bug?

Set rule 'declaration-property-value-no-unknown': null, but disable is not an options because it makes sense. 

Something like that: 
"declaration-property-value-no-unknown": [
      true,
      {
        "ignoreValues": ["/^\\$/"]
      }
]

**Comment by jeddy3** [comment]
@Andy-QT Thanks for the report and for using the template.

You should extend [`stylelint-config-standard-scss`](https://www.npmjs.com/package/stylelint-config-standard-scss) when linting SCSS, rather than extend `stylelint-config-standard` which is for CSS.

---

See https://github.com/stylelint/stylelint/issues/8302#issuecomment-2587051700.

The `declaration-property-value-no-unknown` rule flags values unknown to the W3C CSS specifications. Dollar variables are an SCSS-only construct. From [the rule docs](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown):

> This rule is only appropriate for CSS. You should not turn it on for CSS-like languages, such as SCSS or Less.

Four of the new rules we turned on in the latest version of our standard config are for CSS only. If you're linting SCSS, you should turn them off in your config.

Rather than do it yourself, we recommend in our [getting started guide](
https://stylelint.io/user-guide/get-started#linting-css-like-languages-and-css-within-containers) extending the [shared config created by the SCSS community](https://www.npmjs.com/package/stylelint-config-standard-scss). 

They configure our built-in rules (which are written for CSS) for maximum compatibility with SCSS. When they publish their next version, it will consider these new rules: see https://github.com/stylelint-scss/stylelint-config-recommended-scss/issues/336.

The SCSS community can also investigate maintaining their own version of the rule, as [they already do with several other rules](https://github.com/stylelint-scss/stylelint-scss?tab=readme-ov-file) such `function-no-unknown`, using CSSTree's [syntax extension](https://github.com/csstree/stylelint-validator?tab=readme-ov-file#syntaxextensions). SCSS syntax is complex, and the SCSS community will have to determine whether supporting this type of rule is viable.

**Comment by jeddy3** [comment]
I've just realised the [config example in our docs is wrong](https://stylelint.io/user-guide/get-started#linting-css-like-languages-and-css-within-containers). 

That's on me: https://github.com/stylelint/stylelint/pull/8237. Sorry about that.
